doc/usecases/elektra_web/UC_add_instance.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Add instance
+
+## Summary
+
+Title: Add instance
+Scope: Setup
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User sets up elektrad on the instance and connects it to clusterd according to the corresponding README.
```

Comment:
Try to be a bit more high-level, do not mention elektrad or similar.

Actually "add instance" and "add cluster" should be merged to "describe network topology"